lib/fizzbuzz.rb

```diff
@@ -3,6 +3,15 @@ class SuperFizzBuzz
   def run(input)
 
     #Implement your code here
+		if (input % 15 == 0)
```

Comment:
Take a look at this.
http://www.skorks.com/2009/08/how-a-ruby-case-statement-works-and-what-you-can-do-with-it/

There's no need for parentheses or explicit return statements